src/app/zap-templates/zcl/data-model/chip/application-basic-cluster.xml

```diff
@@ -28,24 +28,19 @@ limitations under the License.
     <attribute side="server" code="0x0001" define="APPLICATION_VENDOR_ID" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">vendor id</attribute>
     <attribute side="server" code="0x0002" define="APPLICATION_NAME" type="CHAR_STRING" length="32" writable="false" optional="false">application name</attribute>
     <attribute side="server" code="0x0003" define="APPLICATION_PRODUCT_ID" type="INT16U" min="0x0000" max="0xFFFF" writable="false" optional="false">product id</attribute>
-    <attribute side="server" code="0x0004" define="APPLICATION_APP" type="ApplicationBasicApp" writable="true" optional="false">application app</attribute>
-    <attribute side="server" code="0x0005" define="APPLICATION_STATUS" type="ENUM8" min="0x00" max="0xFF" default="0x01" writable="false" optional="false">application status</attribute>
+    <attribute side="server" code="0x0004" define="APPLICATION_APP" type="Application" writable="true" optional="false">application app</attribute>
```

Comment:
The name of the attribute here does not match the spec.  Followup to fix?

Reply:
By name of the attribute do you mean both the value inside define="" and the value of attribute tag. In this case define has the value APPLICATION_APP and the attribute tag has application app?

FYI: The issue https://github.com/project-chip/connectedhomeip/issues/13213

Reply:
I mean the text inside the attribute tag.  The define="" is pretty much irrelevant for anything except the old-style attribute ID constants, which we should not be using anyway,

Reply:
Aha, ok. Thanks!